littleblog/entry/serializers.py

```diff
@@ -58,13 +58,25 @@ class Meta:
         fields = ('id', 'title', 'text', 'author', 'preview_image', 'topic',)
 
 
-class RateUpdateSerializer(serializers.ModelSerializer):
+class RatingUpdateSerializer(serializers.ModelSerializer):
+
+    mark = serializers.IntegerField(required=True, write_only=True)
+    current_rating = serializers.FloatField()
+    amount_of_marks = serializers.IntegerField()
+
+    def validate_mark(self, value):
+        if value < 1 or value > 5:
+            raise serializers.ValidationError('Mark value must be between 1 and 5.')
+        return value
```

Comment:
1. drf has something for your case. 

```
mark = serializers.IntegerField(min_value=..., max_value=..., ...)
```

2. https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad -

Reply:
But how can I set my own error text now?  Probably it doesn't matter and it's not needed in this simple case...

Reply:
Yes, in our case custom error message is not a requirement. But, for the future:
![Screenshot 2022-05-09 at 15 20 59](https://user-images.githubusercontent.com/74052345/167408660-948083fd-f546-41b7-96a8-2e2edc7c78aa.png)

You can pass `error_messages` parameter in field definition. Like this `error_messages={"min_value": "Mark value must be bigger than {min_value}"}`. 
However, your current coded uses two values in error messages. To implement it like that you would need to override IntegerField with custom code and handle error message in any way you want